applications/MeshingApplication/custom_processes/parmmg/pmmg_process.h

```diff
@@ -0,0 +1,344 @@
+// KRATOS  __  __ _____ ____  _   _ ___ _   _  ____
+//        |  \/  | ____/ ___|| | | |_ _| \ | |/ ___|
+//        | |\/| |  _| \___ \| |_| || ||  \| | |  _
+//        | |  | | |___ ___) |  _  || || |\  | |_| |
+//        |_|  |_|_____|____/|_| |_|___|_| \_|\____| APPLICATION
+//
+//  License:         BSD License
+//                   license: MeshingApplication/license.txt
+//
+//  Main authors:    Marc Nunez
+//                   Carlos Roig
+//                   Vicente Mataix Ferrandizz
+//
+
+#if !defined(KRATOS_PMMG_PROCESS)
+#define KRATOS_PMMG_PROCESS
+
+// System includes
+
+// Project includes
+#include "processes/process.h"
+#include "custom_utilities/parmmg/pmmg_utilities.h"
+
+// NOTE: The following contains the license of the PMMG library
+/* =============================================================================
+**  Copyright (c) Bx INP/Inria/UBordeaux, 2017- .
+**
+**  parmmg is free software: you can redistribute it and/or modify it
+**  under the terms of the GNU Lesser General Public License as published
+**  by the Free Software Foundation, either version 3 of the License, or
+**  (at your option) any later version.
+**
+**  parmmg is distributed in the hope that it will be useful, but WITHOUT
+**  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+**  FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+**  License for more details.
+**
+**  You should have received a copy of the GNU Lesser General Public
+**  License and of the GNU General Public License along with mmg (in
+**  files COPYING.LESSER and COPYING). If not, see
+**  <http://www.gnu.org/licenses/>. Please read their terms carefully and
+**  use this copy of the mmg distribution only if you accept them.
+** =============================================================================
```

Comment:
why do we need to include this license?
all we do is to create a wrapper no? (not the expert here)

Reply:
We have it as well in mmg_utils and mmg_procss, so I did the same here. Not sure either if it is mandatory.